AudiusProject/audius-protocol
# PLAT-686: identity relays

### Description
Latency calculations for a relay request and its transaction submissions are now included in the redis log record. With the correct health check query params the most recent relay latencies will be reported.

 
### Tests
Tested on stage. Steps to repro:
- run the latest commit from this PR on stage
- go to staging.audius.co
- "heart" or "repost" your favorite track
- check the console to see that `/relay` was done successfully
- go check https://identityservice.staging.audius.co/health_check/relay?verbose=true and you should see a few transactions that you just did
- there should be new latency fields for nethermind and POA reported for each transaction
- you can also add a `maxRelayLateny` seconds query param which will report an error in the health check if any of the transactions present lasted too long


### Monitoring - How will this change be monitored? Are there sufficient logs / alerts?
This is part of the health check so we should be seeing new data in the health monitor. There also was a log added in the poa relay that logs how long the tx took to submit. This is so it has a matching log with the existing nethermind relay.
